test/integration/services/data/test-get-workload-points.js

```diff
@@ -16,8 +17,8 @@ describe('services/data/get-workload-points', function () {
       })
   })
 
-  xit('retrieves the latest points configuration', function (done) {
-    getWorkloadPoints().then(function (result) {
+  xit('retrieves the latest points configuration (non-t2a)', function (done) {
```

Comment:
Since you are updating / adding to these tests anyway, would be worth removing the skip from the test and getting these to pass. See issue #1123. Currently, your test isn't actually running